### PR TITLE
Support create token command

### PR DESCRIPTION
**Pull request by zymap**
This pull request is base on #145, so please merge the previous pull request at first.

*Motivation*

Support create a token using secret key and private key.

*Modifications*

- Add command `create`
- Add test for the command `create`

*Command output*
```
➜  pulsarctl-yong git:(create-token) ./pulsarctl token create -h
USED FOR:
    This command is used for create a token string.

REQUIRED PERMISSION:
    This command does not need any permission.

OUTPUT:
    #Create a token successfully.
    eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJoZWxsby10ZXN0In0.qxaczygeZaZDlK7jQHHXCaQRbwd2wxIHjCH3y_Lo2Q4

    #None of the signing keys is specified.
    [✖]  none of the signing keys is specified

    #Signing key is specified more than one.
    [✖]  the signing key is specified more than one

Usage: pulsarctl token create [flags]

Examples:
    #Create a token using a secret key string.
    pulsarctl token create --secret-key-string (secret-key-string) --subject (subject)

    #Create a token using a secret key file.
    pulsarctl token create --secret-key-file (secret-key-file-path) --subject (subject)

    #Create a token using a private key file.
    pulsarctl token create --private-key-file (private-key-file-path) --subject (subject)

    #Create a token with expire time.
    pulsarctl token create --secret-key-string (secret-key-string) --subject (subject) --expire 1m

    #Create a token using a base64 encoded secret key.
    pulsarctl token create --secret-key-string (secret-key-string) --base64 --subject (subject)



Create a token flags:
  -a, --signature-algorithm string   The signature algorithm used to generate the secret key or the private key, generate a token needs to use the same signature algorithm. Valid options are: 'HS256', 'HS384', 'HS512', 'RS256', 'RS384', 'RS512', 'PS256', 'PS384', 'PS512', 'ES256', 'ES384', 'ES512'. (default "RS256")
      --secret-key-string string     The secret key string that used to sign a token.
      --secret-key-file string       The secret key file that used to sign a token.
      --private-key-file string      The private key file that used to sign a toke.
      --subject string               The 'subject' or 'principal' associate with this token.
      --expire string                The expire time for a token. e.g. 1s, 1m, 1h
      --base64                       The secret key is base64 encoded or not.
```

### CODE REVIEW

**Comment by sijie** [comment]
> This pull request is base on #145, so please merge the previous pull request at first.

#145 is still WIP?

**Comment by sijie** [comment]
@zymap can you rebase this pull request?

**Comment by zymap** [comment]
@sijie I rebase this pull request. PTAL. Thanks.

File: pkg/ctl/token/create_test.go [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+	"RS512",
+	"ES256",
+	"ES384",
+	"ES512",
+	"PS256",
+	"PS384",
+	"PS512",
+}
+
+var keyFiles = []string{
```
Comment (sijie) [inline pkg/ctl/token/create_test.go]: Can you move these key files to a different folder? I don't think it is a good idea to keep them in `pkg` folder.

Comment (zymap) [reply pkg/ctl/token/create_test.go]: Ok. I move the keys to the test directory.

**Comment by zymap** [comment]
@sijie PTAL. Thanks.

**Pull request merged**